docs/docs/persistent-volumes.md

```diff
@@ -0,0 +1,91 @@
+1. Specify the container path, from which your application will read and write data. The container path must be non-nested and cannot contain slashes e.g. `data`, but not  `../../../etc/opt` or `/user/data/`.
+1. Click Create.
+
+**Note:** After you launch your application, you will not be able to change the specifications for your local persistent volumes.
+
+<!--
+<screen shot>
+-->
+
+Alternatively, you can configure your application to use persistent local volumes in your application definition JSON file:
+
+<pre>
+```
+{
+  "id": "/sesame/samson",
+  "residency": {
+    "relaunchEscalationTimeoutSeconds": 30,
+    "taskLostBehavior": "relaunchAfterTimeout"
+  },
+  "cpus": 0.5,
+  "mem": 512.0,
+  "disk": 1024.0,
+  "instances": 2,
+  "container": {
+    "type": "MESOS",
+    <b>"volumes": [
+      {
+        "containerPath": "data",
+        "mode": "RW",
+        "persistent": {
+          "size": 1234
+        } </b>
+      }
+    ]
+  }
+}
+```
+</pre>
+
+## View the Status of Your Application with Persistent Local Volumes
+
+After you have created your Marathon application, click on the “Volumes” tab of the application detail view [check that vocab] to get detailed information about your app instances and associated volumes.
+
+The Status column tells you if your app instance is attached to the volume or not. The app instance will read as “detached” if you have scaled down your application. Currently the only Operation Type available is read/write (RW).
+
+<!--
+<screen shot>
+-->
+
+Clicking on the individual volume brings you to the Volume Detail Page, where you can see information about the individual volume:
+
+<!--
+<screen shot>
+-->
+
+## The Mesos Sandbox
+
+While persistent local volumes allow your application to read and write data to a particular node or nodes, the temporary Mesos sandbox is still the target for the `stdout` and `stderr` logs. To view these logs, go to the Marathon pane of the DCOS web interface.
+
+## Potential Pitfalls
+
+### Replication and Backups
+
+Because tasks are pinned to nodes, your data can be lost if the node is lost. If you need to guard against data loss, you must set up replication and backups. 
+
+### Non-unique Roles
+
+In Mesos, volumes are created for *roles*, not for frameworks. If more than one framework is registered with a particular role, the resource offer meant for Marathon may be accepted by another framework. This can result in volume creation failure. Make sure you register Marathon with a unique role that is not used by another framework instance. This also holds true for any other Marathon instance you are running Multiple Marathon instances in HA mode do not need to have unique roles, however, because they are considered one instance.
```

Comment:
I think you're missing a period between `... you are running` and `Multiple Marathon ...`.

Also, better change `they are considered one instance` to `they are meant to use the same role`, as the former may be confusing. Sorry in case I proposed that in an earlier comment.